.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,50 @@
+name: GitHub Pages
+
+on:
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  docs:
+    name: "Deploy"
+    runs-on: ubuntu-latest
```

Comment:
For reproducibility, I suggest to use `ubuntu-18.04`  or `ubuntu-20.04` .

Reply:
Done in https://github.com/robotology/osqp-eigen/pull/65/commits/1e3ed3badd2bebcf961e7a46624d0a7a69bef0bd